dev/com.ibm.websphere.security/src/com/ibm/websphere/security/audit/AuditEvent.java

```diff
@@ -114,6 +114,15 @@
     public static final String TARGET_EJB_BEAN_NAME = "target.ejb.beanname";
     public static final String TARGET_EJB_METHOD_PARAMETERS = "target.ejb.method.parameters";
 
+    public static final String TARGET_SAF_RETURN_CODE = "target.saf.return.code";
```

Comment:
@emilytee Is the `target` prefix appropriate for return and reason codes that are coming back, and explain the authorization decision?

Reply:
Typically we put the return and reason codes in AuditEvent.REASON_CODE and AuditEvent.REASON_TYPE, and that's consistent across all of the events.  However, if there's some additional fine-grained code that is coming from SAF that is also relevant to save, you can add the target prefix to a new field,  like you did for TARGET_SAF_REASON_CODE

Reply:
ok thanks - I have no issues here.